adcc/testdata/psi_dump.yml

```diff
@@ -0,0 +1,12 @@
+formaldehyde_sto3g_pcm_adc1:
```

Comment:
Can you maybe add the Python file with which you created the reference data? 😄

Reply:
You mean add it to the testdata folder?
Sure. But why?

Reply:
For reproducibility purposes, and in case somebody except yourself wants to generate more reference data with the same setup. I think there is also a script file that generates Q-Chem data or so. You could create a `generate_psi4_data.py` script that produces your .yml file 😃